CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Review/ReviewViewController.swift

```diff
@@ -306,10 +306,17 @@ extension ReviewViewController {
             }
             navigationBarBottomAdapter?.setMainButtonClickedActionCallback { [weak self] in
                 guard let self = self else { return }
+                let eventProperties = [AnalyticsProperty(key: .numberOfPagesScanned,
+                                                         value: pages.count)]
+
+                AnalyticsManager.track(event: .processTapped,
+                                       screenName: .review,
+                                       properties: eventProperties)
```

Comment:
[suggestion]: I suggest extracting this logic in a separate method and use it in both places for normal navigation and this bottom bar navigation.
Please follow the naming for this method like:
```
 private func sendAnalyticsScreenShown() {
 ```